PennMobile/src/main/java/com/pennapps/labs/pennmobile/MainActivity.kt

```diff
@@ -77,6 +81,18 @@ class MainActivity : AppCompatActivity() {
     private lateinit var mFirebaseAnalytics: FirebaseAnalytics
     val mNetworkManager by lazy { OAuth2NetworkManager(this) }
 
+    // Declare the launcher at the top of your Activity/Fragment:
+    private val requestPermissionLauncher =
+        registerForActivityResult(
+            ActivityResultContracts.RequestPermission(),
+        ) { isGranted: Boolean ->
+            if (isGranted) {
+                // FCM SDK (and your app) can post notifications.
```

Comment:
If isGranted is true, does that just mean you can send notifications even though nothing is written here.

Reply:
doubling this question

Is this something where we can choose to do something (ex: sending a "success!" message to the user), but it's also okay not to do anything (since notifications will be shown as enabled in the background?)

Reply:
I don't think we have to do anything, but having TODOs in main is generally considered bad practice. What behavior should we expect if notifications aren't enabled? Is this information generally available or should it be stored somewhere?